docs/docs/plugin-authoring.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Plugin Authoring
+---
+
+Creating custom plugins in Gatsby is a breeze with our straightforward authoring process.
```

Comment:
We've been following @robwierzbowski's suggestion to avoid indicating something is "easy" as that can intimidate newer developers for whom everything is complicated

https://github.com/gatsbyjs/gatsby/issues/3159

Reply:
Agreed. Another way to word this could be:
>The Gatsby API is open and flexible and the number of plugins created by community members is growing. Follow these steps to create a custom plugin in Gatsby.

I don't know if that is gobbledygook language since I'm not a programmer. The essence would be to emphasize how Gatsby is meant to be open to plugins. It was built to expect them. Or something like that. Just a tone that invites people to experiment and realize that there is a lot of potential for an awesome plugin ecosystem.

Reply:
Hey, thanks for remembering! My two cents:

> Gatsby is designed to be extensible. One of the best ways to add functionality to Gatsby is through our plugin system. (maybe some examples of what is a good fit for a plugin...)

Reply:
Good call. I'm leaning toward @robwierzbowski's wording, and I'll work in some examples of good candidates for plugins.